content/references/http-websocket-apis/public-api-methods/clio-methods/nft_history.md

```diff
@@ -0,0 +1,706 @@
+| `limit`        | UInt32 | _(Optional)_ Limit the number of NFTs to retrieve. The server is not required to honor this value. |
+| `marker`       | Marker | Value from a previous paginated response. Resume retrieving data where that response left off. This value is NOT stable if there is a change in the server's range of available ledgers. If you are querying the “validated” ledger, it is possible that new NFTs are created during your paging. |
+
+**Note** If you do not specify a ledger version, Clio uses the latest validated ledger.
+
+## Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 0,
+  "type": "response",
+  "result": {
+    "ledger_index_min": 21377274,
+    "ledger_index_max": 27482261,
+    "transactions": [
+      {
+        "meta": {
+          "AffectedNodes": [
+            {
+              "DeletedNode": {
+                "FinalFields": {
+                  "Flags": 1,
+                  "NFTokenID": "00082710A955EBAD4AE6261466FBEEEBF6A59591AC8C831F16E5DA9C00000001",
+                  "RootIndex": "33A64DE4A5B00FDB23036216653F1059C5ED7546E502DE878500DF667D61551E"
+                },
+                "LedgerEntryType": "DirectoryNode",
+                "LedgerIndex": "33A64DE4A5B00FDB23036216653F1059C5ED7546E502DE878500DF667D61551E"
+              }
+            },
+            {
+              "ModifiedNode": {
+                "FinalFields": {
+                  "Account": "rGS43H57KGnvsDapj796XXo3g5qWcQxqmA",
+                  "Balance": "1079999928",
+                  "BurnedNFTokens": 1,
+                  "Flags": 0,
+                  "MintedNFTokens": 2,
+                  "OwnerCount": 0,
+                  "Sequence": 27479932
+                },
+                "LedgerEntryType": "AccountRoot",
+                "LedgerIndex": "6745C90D426A198852939F3F3B97C8467C16147969BF68C773702BED477F4C1B",
+                "PreviousFields": {
+                  "Balance": "979999940",
+                  "OwnerCount": 1,
+                  "Sequence": 27479931
+                },
+                "PreviousTxnID": "377B9A2DBA9448911BDC3C241104C2F6BAE0BF62A2E32976E0FCE33ED945EA8E",
+                "PreviousTxnLgrSeq": 27481755
+              }
+            },
+            {
+              "DeletedNode": {
+                "FinalFields": {
+                  "Amount": "100000000",
+                  "Flags": 0,
+                  "NFTokenID": "00082710A955EBAD4AE6261466FBEEEBF6A59591AC8C831F16E5DA9C00000001",
+                  "NFTokenOfferNode": "0",
+                  "Owner": "rHKjdRD2aHzEojrioJb4tw6p4i7FS1hUkc",
+                  "OwnerNode": "0",
+                  "PreviousTxnID": "B2E36CBAE4C0E329A0FA373DBC853AE871F9EC0371C6299FE0955F689671F015",
+                  "PreviousTxnLgrSeq": 27481778
+                },
```

Comment:
wondering whether it makes sense to use an example with the `binary` option set, to save space. You could then show an additional example with the `binary` flag not set but with only one transaction to show what that format looks like.